<launch>
    <node pkg="ros_tutorial" type="controller.py" name="controller" output="screen">
        <param name="linear_speed" value="0.6" />
        <param name="angular_speed" value="+0.1"/>
        <param name="goal_angle" value="93"/>
        <param name="epsilon" value="0.001"/> 
        <param name="length" value="3"/>
        <param name="width" value="2"/>   
    </node>

    <node pkg="ros_tutorial" type="monitor.py" name="monitor"></node>

    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
        <arg name="x_pos" value="0.0"/>
        <arg name="y_pos" value="0.0"/>
        <arg name="z_pos" value="0.0"/>
    </include>

    include file="$(find turtlebot_rviz_launchers)/launch/view_robot.launch"/>

   
</launch>
